recipes/galpy/meta.yaml

```diff
@@ -15,7 +15,7 @@ source:
 build:
   number: 0
   script: python setup.py install --single-version-externally-managed --record=record.txt  # [win]
-  script: CFLAGS="$PREFIX/include/gsl" python setup.py install --single-version-externally-managed --record=record.txt  # [not win]
+  script: CPATH="$PREFIX/include/gsl" python setup.py install --single-version-externally-managed --record=record.txt  # [not win]
```

Comment:
The `/gsl` part is going to break this too. Maybe `CPATH="$PREFIX/include:$CPATH"` would be best.